tests/rptest/tests/restart_services_test.py

```diff
@@ -67,3 +67,10 @@ def test_restart_services(self):
         check_service_restart(self.redpanda, "Restarting the schema registry")
         self.logger.debug(result_raw)
         assert result_raw.status_code == requests.codes.ok
+
+        self.logger.debug("Check health monitor reset")
+        result_raw = admin.reset_health_info()
+        self.logger.debug(result_raw)
+        check_service_restart(self.redpanda,
+                              "Resetting Health Monitor Frontend")
+        assert result_raw.status_code == requests.codes.ok
```

Comment:
@mmaslankaprv @VadimPlh Do you have any recommendation on a better test for this? I would like to include a DT test that
1. Does something with a cluster such that the health monitor has some state
2. Reset the health monitor with `POST /v1/debug/reset_health_info`
3. Check that the health monitor has valid state

Reply:
So you need to test that after you restart the health_monitor_frontend redpanda will update `_cluster_disk_health`